File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = set() # 중복을 허용하지 않는 set 자료형 사용
    
    # 모든 수의 조합에 대해 합을 구하고 set에 추가
    for i in range(len(numbers)):
        for j in range(i+1, len(numbers)):
            answer.add(numbers[i] + numbers[j])
    
    return sorted(list(answer))

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    numbers.sort((a,b)=>a-b);
    
    let result = [];
    
    for(let i=0; i<numbers.length; i++){
        for(let j=i+1; j<numbers.length; j++){
            if(!result.includes(numbers[i] + numbers[j])){
                result.push(numbers[i] + numbers[j]);
            }
        }
    }
    result.sort((a,b)=>a-b)
    return result;
}